DOCKER.md

```diff
@@ -49,3 +49,25 @@ You can have the gateway run in the background by using a `-d` or `--daemon` arg
 
 Once the gateway runs in the background with `docker-run`, you can interact with its REST server on `http://localhost:8080`, and the serial port
 of the node is redirected on `localhost` TCP socket on port `20000`.
+
+## Running under macOS
+
+Docker for Mac does not support adding a host device to a container (e.g. `docker run --device`). [source](https://docs.docker.com/docker-for-mac/faqs/#can-i-pass-through-a-usb-device-to-a-container)  
+The use of [Docker Toolbox](https://docs.docker.com/toolbox/overview/) is recommended.
+
+Start Docker Toolbox with the Docker Quickstart Terminal, then do this additionnal steps on the boot2docker VM used by Docker Toolbox:
+1. Install udev rules.
+
+        $ docker-machine ssh
+        docker@default:~$ git clone https://www.github.com/iot-lab/iot-lab-gateway.git && cd iot-lab-gateway
+        docker@default:~$ sudo cp bin/rules.d/* /etc/udev/rules.d/.
```

Comment:
@vandaele  Can we simply use python setup.py udev_rules_install here ? Hmm, maybe the boot2docker has no python

Reply:
That's it

    docker@default:~$ python
    -sh: python: not found